test/val/val_type_unique_test.cpp

```diff
@@ -238,4 +238,57 @@ OpMemoryModel Physical32 OpenCL
               Not(HasSubstr(GetErrorString(SpvOpTypeVoid))));
 }
 
+TEST_F(ValidateTypeUnique, PointerTypesSameArrayStrideNoExtension) {
```

Comment:
The test name is misleading to me:  The array strides here are different, but the test name is ....SameArrayStride...

Reply:
CopyPaste